client/api_object.go

```diff
@@ -256,6 +262,33 @@ func (c *client) PutObject(ctx context.Context, bucketName, objectName string, o
 	return nil
 }
 
+func (c *client) headSPObjectInfo(ctx context.Context, bucketName, objectName string) error {
+	_, err := c.HeadObject(ctx, bucketName, objectName)
+	if err != nil {
+		return fmt.Errorf("fail to get object info on chain: %s", err.Error())
+	}
```

Comment:
I think this is unnecessary, no need to query this info from the user side again.

Reply:
removed